venues/ICLR.cc/2018/Conference/python/correct-authorgroups.py

```diff
@@ -0,0 +1,80 @@
+import openreview
+import config
+import csv
+import argparse
+from collections import defaultdict
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print client.baseurl
+
+# get all the authorgroups.
+# where does 1176 come from?
+# 1176 is the value on the counter for ICLR.cc/2018/Conference/-/Submission
+authorgroups = []
+print "finding valid authorgroups ..."
+for i in range(1, 1176):
+    groupid = 'ICLR.cc/2018/Conference/Paper{0}/Authors'.format(i)
```

Comment:
why don't you use this call to get all the existent groups?

https://openreview.net/groups?id=ICLR.cc/2018/Conference/Paper.*/Authors

Reply:
ah, i forgot about this. i can change it